docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```diff
@@ -0,0 +1,152 @@
+---
+title: How to create example cases for a Custom Expectation
+---
+import Prerequisites from '../creating_custom_expectations/components/prerequisites.jsx'
+
+This guide will help you add example cases to document and test the behavior of your [Expectation](../../../reference/expectations/expectations.md). 
+
+<Prerequisites>
+
+ - Created a [Custom Expectation](../creating_custom_expectations/overview.md)
+
+</Prerequisites>
+
+
+Example cases in Great Expectations serve a dual purpose:
+* They help the users of the Expectation understand its logic by providing examples of input data that the Expectation will evaluate;
+* And provide test cases that the Great Expectations testing framework can execute automatically.
+
+If you decide to contribute your Expectation, its entry in the [Expectations Gallery](https://greatexpectations.io/expectations/) will render these examples.
+
+We will explain the structure of these tests using the Custom Expectation implemented in [How to create Custom Column Aggregate Expectations](../creating_custom_expectations/how_to_create_custom_column_aggregate_expectations.md).
+
+### Steps
+
+#### 1. Decide which tests you want to implement
```

Comment:
```suggestion
## Steps

### 1. Decide which tests you want to implement
```
As a rule of thumb we want our headings to nest from H1 > H2 > H3. Since the page's title is treated as our H1, the next major subsection should be at the H2 level, and the subsections under that should be at H3.